packages/material-ui/src/TablePagination/TablePagination.js

```diff
@@ -100,7 +100,6 @@ const TablePagination = React.forwardRef(function TablePagination(props, ref) {
     ...other
   } = props;
 
-  const onChangePage = onChangePageProp || onPageChangeProp;
   const onChangeRowsPerPage = onChangeRowsPerPageProp || onRowsPerPageChangeProp;
```

Comment:
I would expect the same issue with these two props, even if less frequent. Should we apply the same strategy?

Reply:
@oliviertassinari If you have a concrete problem, then please open a separate issue. I don't understand from your statement alone what the problem with the other prop is.

Reply:
@eps1lon Ok, my bad, TablePagination doesn't forward `onChangeRowsPerPage` to `ActionsComponent`.